plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestParquetPageSkippingWithAcceleratedReader.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.QueryRunner;
+
+public class TestParquetPageSkippingWithAcceleratedReader
```

Comment:
are there there other tests like this that require abstraction?

Reply:
TestTimestampMicros, TestTimestamp, TestParquetDecimalScaling and TestFullParquetReader are also updated to run the new code in slightly different ways depending on what makes more sense in the context of those tests. If you see other tests which need update, please let me know.